src/common/serverinfo.ts

```diff
@@ -29,7 +29,8 @@ export type GetServerInfoResponse = {
     reserveIncrementXRP: string,
     ledgerVersion: number
   },
-  validationQuorum: number
+  validationQuorum: number,
+  networkLedger?: string
```

Comment:
Looks like we were testing for this value but hadn't covered in in our type definitions. Adding it here as an optional property.